exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/ProfileResources.java

```diff
@@ -68,20 +68,35 @@
   @Inject DrillUserPrincipal principal;
   @Inject SecurityContext sc;
 
+  public static String getPrettyDuration(long durationInMillis) {
+    long hours = TimeUnit.MILLISECONDS.toHours(durationInMillis);
```

Comment:
May be this utility method can accept two parameters: `long start, long end`, so you won't have to repeat the following logic `profile.getEnd()-profile.getStart()` or `endTime - startTime`?